pom.xml

```diff
@@ -4,6 +4,13 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
+    <parent>
+        <groupId>io.nats</groupId>
+        <artifactId>nats-parent</artifactId>
+        <version>1.0-SNAPSHOT</version>
+<!--         <relativePath>../nats-parent-pom/pom.xml</relativePath> -->
```

Comment:
No, good catch. I had meant to delete this. I'll fix it!

On Friday, April 15, 2016, Colin Sullivan <notifications@github.com> wrote:

> In pom.xml
> <https://github.com/nats-io/nats-connector-framework/pull/27#discussion_r59891382>
> :
>
> > @@ -4,6 +4,13 @@
> >           xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
> >      <modelVersion>4.0.0</modelVersion>
> >
> > +    <parent>
> > +        <groupId>io.nats</groupId>
> > +        <artifactId>nats-parent</artifactId>
> > +        <version>1.0-SNAPSHOT</version>
> > +<!--         <relativePath>../nats-parent-pom/pom.xml</relativePath> -->
>
> Any reason to keep this here? Debugging?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/nats-io/nats-connector-framework/pull/27/files/1e36eb52a2e11587c0505e80ff2a69db8375d84e#r59891382>
>


-- 
*Larry McQueary* | Director, Messaging Technology
larry@apcera.com | @mcqueary | github.com/mcqueary